src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a class="nav__link-item first is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Smartphones</a>
+          </li>
+          <li class="nav__link">
+            <a
+              class="nav__link-item"
+              href="#"
+              data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Gadgets</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Tablets</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Photo</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__link-item" href="#">Video</a>
+          </li>
+        </ul>
```

Comment:
Add spaces between multiline **siblings**, check [the list here](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)